ci/sonar-scanner.sh

```diff
@@ -66,9 +66,11 @@ pushd "workers/unity"
     # Wildcards don't appear to play nice with this.
     args+=("-d:sonar.cs.opencover.reportsPaths=$(find ./logs -name "*.xml" | paste -sd "," -)")
 
+    dotnet new globaljson --sdk-version 2.2.402 --force
```

Comment:
Which of the 3 dotnet runs below require 2.2?
If it's only the middle one, could the override not be added to that execution specifically?

Reply:
What do you mean? Put the `new globaljson` command after the first `dotnet-sonarscanner begin`?

Reply:
I mean running `dotnet msbuild --runtime 2.2.402 ...` instead of the global override

Reply:
I tried but couldn't find a way to do that. Everywhere I looked just referred to that json file :(